monarchmoney/monarchmoney.py

```diff
@@ -307,6 +307,46 @@ async def create_manual_account(
             variables=variables,
         )
 
+    async def delete_account(
```

Comment:
same as above, please stack these commits atop origin/main, instead of the other branch you've created for #85

Reply:
done!